### ISSUE TITLE
Highlight on :s command

### ISSUE THREAD

**Issue by korgoth** [NeoVintageous/NeoVintageous#992]
Is it possible to highlight the matched test when using the :s command? 

When using search with / - the matched text is automatically highlighted. Can this be done when using :s/<search_string> too? 

Would be very helpful, but i cant figure out how to do it.

Thank!

**Comment by gerardroche** [comment]
Yes. Realtime substitutions. I've seen that feature in Neovim. It's awesome. Neovim even shows the replacements in realtime. Really cool. It's on the top of the roadmap.
